### DISCUSSION TITLE
Intel Meteorlake has just launched and has built in NPU

### DISCUSSION THREAD

**Discussion by twctg81** [blakeblackshear/frigate#9038] [General]
Anyone planning to get it and test it with Frigate?
Just saw this in Tomshardware, and it seems like processors in Meteorlake onwards would have built in NPU. 
Does anyone know if these new NPU would mean that inference speed will be in google coral range ?  

https://www.tomshardware.com/laptops/i-tested-intels-meteor-lake-cpus-on-ai-workloads-and-amds-chips-sometimes-beat-them
![VZwiVwsYZsv9589LA6QUvT](https://github.com/blakeblackshear/frigate/assets/68601394/94db1c15-be91-4ccd-97ce-9764a85fe594)

**Comment by ghost** [comment]
These have higher numbers (TOPS) than coral, so maybe even better?

**Comment by philippedegeer** [comment]
I am planning to buy a Shuttle PC with a Core Ultra 7 when they release it in Q3. It seems that Openvino is supporting the NPU but I am not certain Frigate is on the latest version of Openvino. Nevertheless, based on what I achieve with the iGPU of a Celeron 5105 (openvino - yolov8s 320 - 30ms inference time), I expect the performance of the Core Ultra Arc iGPU to be in the Coral range.
